pkg/sql/schemachanger/scplan/internal/opgen/opgen_secondary_index.go

```diff
@@ -86,6 +86,14 @@ func init() {
 				}),
 			),
 			to(scpb.Status_PUBLIC,
+				emit(func(this *scpb.SecondaryIndex) *scop.RefreshStats {
+					return &scop.RefreshStats{
+						TableID: this.TableID,
+					}
+				}),
+				emit(func(this *scpb.SecondaryIndex, md *opGenContext) *scop.LogEvent {
+					return newLogEventOp(this, md)
+				}),
```

Comment:
I am curious is there a reason that we need to emit those two operations before `MakeValidatedSecondaryIndexPublic`? Or it does not matter?

Reply:
It honestly doesn't matter in this case, since if the later operation fails none of these will do anything either. These commands only have a side effect once a stage has finished execution.